modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/InternalCellRenderer.ts

```diff
@@ -49,6 +53,12 @@ export const INTERNAL_CELL_RENDERERS: Array<IInternalRenderer> = [
 		name: 'actionLink',
 		type: 'internal',
 	},
+	{
+		component: ArrayRenderer,
+		label: Liferay.Language.get('array'),
+		name: 'array',
+		type: 'internal',
```

Comment:
This makes ArrayRenderer to be listed in the DSM as a valid choice for rendering a field. This is not bad neither good, but we need to consider if we want to expose it in the DSM as any other or prefer to use it under the hoods in the FDS when field comes in the shape of an array. Again internal discussions are taking place